modules/apps/commerce/commerce-product-service/src/main/java/com/liferay/commerce/product/service/impl/CPConfigurationListLocalServiceImpl.java

```diff
@@ -426,9 +426,6 @@ private GroupByStep _getGroupByStep(
 			CPConfigurationListTable.INSTANCE.companyId.eq(companyId)
 		).and(
 			CPConfigurationListTable.INSTANCE.groupId.eq(groupId)
-		).and(
-			CPConfigurationListTable.INSTANCE.masterCPConfigurationList.eq(
-				false)
```

Comment:
@andrea-ale-sbarra since we are including the masterConfigurationList in the returned list, I think it will always return the masterConfiguartionList first. We probably want to add a sort to this so that it would grab any child ones first since eligibility can not be set for masterConfigurationList.

Reply:
Added this to the discovery, https://github.com/dsitu/liferay-portal/commit/d72c6ca98137f8e86d1e940da85c4e690bd2c322